nixos/modules/services/misc/paperless.nix

```diff
@@ -380,6 +390,27 @@ in
         services.paperless.manage = manage;
         environment.systemPackages = [ manage ];
 
+        services.nginx = lib.mkIf cfg.configureNginx {
+          enable = true;
+          upstreams.paperless.servers."${cfg.address}:${toString cfg.port}" = { };
+          virtualHosts.${cfg.domain} = {
+            forceSSL = lib.mkDefault true;
+            locations = {
+              "/".proxyPass = "http://paperless";
```

Comment:
I guess we need `priority` here to ensure the correct order of `locations` in the final `/etc/nginx/nginx.conf`?

Reply:
Isn't the most specific path matching by default?

Reply:
True, I was thinking of regex matching. Nvm as well :slightly_smiling_face: